ui/tests/integration/components/clients/page/sync-test.js

```diff
@@ -26,22 +27,9 @@ module('Integration | Component | clients | Clients::Page::Sync', function (hook
   setupMirage(hooks);
 
   hooks.beforeEach(async function () {
-    clientsHandler(this.server);
```

Comment:
i promise i didn't actually delete this! i just needed to split the tests out into two modules because i needed to use the `syncHandler` for some tests and no handler for another. mirage doesn't behave well if you try to switch between handlers within the same module, nor does it work when you try to overwrite a handler already in use. therefore it was easiest to split these up.